profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info

```diff
@@ -0,0 +1,260 @@
+name = Nexteuropa Newsroom
+description = Integration with the Newsroom application
+core = 7.x
+package = NextEuropa
+// @codingStandardsIgnoreLine
+version = 7.x-2.0
+// @codingStandardsIgnoreLine
+project = nexteuropa_newsroom
+dependencies[] = entity_translation
+dependencies[] = geofield
+dependencies[] = feeds_et
+dependencies[] = feeds
+dependencies[] = feeds_tamper
+dependencies[] = feeds_xpathparser
+dependencies[] = i18n_taxonomy
+dependencies[] = title
+dependencies[] = views_tree
+configure = admin/config/content/newsroom
+feature_sets[nexteuropa_newsroom][title] = Nexteuropa newsroom
```

Comment:
Based on the last meeting with DG COMM and CNECT we won't allow the site administrators to enable the feature via feature set interface, so please remove the affected lines.